Paralayout.podspec

```diff
@@ -8,4 +8,8 @@ Pod::Spec.new do |s|
   s.source   = { :git => 'https://github.com/square/Paralayout.git', :tag => s.version }
   s.source_files = 'Paralayout/*.{swift}'
   s.ios.deployment_target = '9.0'
+
+  s.test_spec 'Tests' do |test_spec|
```

Comment:
pretty slick to see this in action! Wanna add this to Valet? 😉

Reply:
yes and I think I can now with 1.4.0 due to app host support! PR coming up.

Reply:
Done!